cosmwasm/ibc-union/core/msg/src/lightclient.rs

```diff
@@ -32,12 +24,29 @@ pub enum VerifyCreationResponseEvent {
     },
 }
 
+pub type StorageWrites = Vec<(Bytes, Bytes)>;
```

Comment:
we should use a newtype or a map structure (btreemap), some way to enforce no duplicate keys

Reply:
I would say btreemap is the simplest